src/app/rpcserver/fd_rpc_service.c

```diff
@@ -1323,10 +1468,21 @@ method_getTransaction(struct json_values* values, fd_rpc_ctx_t * ctx) {
 // Implementation of the "getTransactionCount" methods
 static int
 method_getTransactionCount(struct json_values* values, fd_rpc_ctx_t * ctx) {
-  (void)values;
-  (void)ctx;
-  fd_webserver_t * ws = &ctx->global->ws;
-  fd_web_error(ws, "getTransactionCount is not implemented");
+  FD_METHOD_SCRATCH_BEGIN( 1<<28 ) { /* read_epoch consumes a ton of scratch space! */
```

Comment:
Use the transaction count from last_notify

Reply:
doesnt work because user passes in commitment level